Formula/youtube-dlc.rb

```diff
@@ -0,0 +1,19 @@
+class YoutubeDlc < Formula
+  desc "Download YouTube videos from the command-line"
+  homepage "https://blackjack4494.github.io/youtube-dlc/"
+  version "2020.09.13"
+  url "https://github.com/blackjack4494/youtube-dlc/releases/download/#{version}/youtube-dlc"
+  sha256 "633caf38c990ab1b21cfe849151601e8a1548ddcdd539caf755e1b3d47941af9"
+  license "Unlicense"
+
+  def install
+    bin.install "youtube-dlc"
+  end
+
+  test do
+    # commit history of homebrew-core repo
```

Comment:
?

Reply:
That weird comment is also frankly in the youtube-dl.rb formula that was modified in this PR.